test/APIConsumer_test.js

```diff
@@ -1,36 +1,29 @@
+const { expect } = require('chai')
 
-const { expect } = require("chai")
-
-describe("APIConsumer", async function () {
+describe('APIConsumer', async function () {
   let apiConsumer, mockOracle, linkToken
+
   beforeEach(async () => {
-    // This runs the deploy script, and deploys everything in deploy!
-    // We have it setup so that we don't deploy mocks
-    // We ONLY deploy the fixtures on a localnet
-    // If we want to deploy fixtures to a testnet, just run `hh deploy --network network`
-    let chainId = await getChainId()
-    if (chainId == 31337) {
-      await deployments.fixture()
-    }
+    await deployments.fixture('api')
```

Comment:
won't this also deploy mocks? We don't always want to do that right?

Reply:
In tests we do.